package/Maps/Yuri's Revenge/2_alpine_assault.map

```diff
@@ -136,7 +630,7 @@ PlayerControl=no
 01000002=1,8,0,0
 
 [FA2spVersionControl]
-Version=881
+Version=884
```

Comment:
what is this version #? for your own reference?

Reply:
FA2SP auto-increments this number when you manually save the map, or when it auto-saves.